fontique/src/backend/fontconfig/config.rs

```diff
@@ -175,10 +175,7 @@ fn include_config(path: &Path, sink: &mut impl ParserSink) -> std::io::Result<()
     Ok(())
 }
 
-fn resolve_dir(
-    node: Node,
-    config_file_path: impl AsRef<std::path::Path>,
-) -> Option<std::path::PathBuf> {
+fn resolve_dir(node: Node, config_file_path: impl AsRef<Path>) -> Option<std::path::PathBuf> {
```

Comment:
There's a consistency argument to make for this remaining as `std::path::Path` and the other usages of that to do the same and remove the `use std::path::Path` since all other `std` stuff comes in via the fully qualified name.

Reply:
IMO we probably ought to go the other way and have `use` imports for everything in `std` (unless there's a name clash)